ERROR - 2023-06-14 00:32:35 --> Error connecting to the database: mysqli_sql_exception: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyectosistema\system\Database\MySQLi\Connection.php:174
Stack trace:
#0 C:\xampp\htdocs\proyectosistema\system\Database\MySQLi\Connection.php(174): mysqli->real_connect('localhost', 'dscriptn_root', 'AkJkSzpj123', 'dscriptn_db_sis...', 3306, '', 0)
#1 C:\xampp\htdocs\proyectosistema\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#2 C:\xampp\htdocs\proyectosistema\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#3 C:\xampp\htdocs\proyectosistema\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 C:\xampp\htdocs\proyectosistema\system\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
#5 C:\xampp\htdocs\proyectosistema\app\Models\Users.php(21): CodeIgniter\Model->__call('get', Array)
#6 C:\xampp\htdocs\proyectosistema\app\Controllers\Login.php(24): App\Models\Users->obtenerUsuario('admin123')
#7 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(934): App\Controllers\Login->acceder()
#8 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#9 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#10 C:\xampp\htdocs\proyectosistema\index.php(67): CodeIgniter\CodeIgniter->run()
#11 {main}

Next CodeIgniter\Database\Exceptions\DatabaseException: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión in C:\xampp\htdocs\proyectosistema\system\Database\MySQLi\Connection.php:218
Stack trace:
#0 C:\xampp\htdocs\proyectosistema\system\Database\BaseConnection.php(391): CodeIgniter\Database\MySQLi\Connection->connect(false)
#1 C:\xampp\htdocs\proyectosistema\system\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
#2 C:\xampp\htdocs\proyectosistema\system\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#3 C:\xampp\htdocs\proyectosistema\system\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
#4 C:\xampp\htdocs\proyectosistema\app\Models\Users.php(21): CodeIgniter\Model->__call('get', Array)
#5 C:\xampp\htdocs\proyectosistema\app\Controllers\Login.php(24): App\Models\Users->obtenerUsuario('admin123')
#6 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(934): App\Controllers\Login->acceder()
#7 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
#8 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#9 C:\xampp\htdocs\proyectosistema\index.php(67): CodeIgniter\CodeIgniter->run()
#10 {main}
CRITICAL - 2023-06-14 00:32:35 --> Unable to connect to the database.
Main connection [MySQLi]: No se puede establecer una conexión ya que el equipo de destino denegó expresamente dicha conexión
in SYSTEMPATH\Database\BaseConnection.php on line 427.
 1 SYSTEMPATH\Database\BaseConnection.php(573): CodeIgniter\Database\BaseConnection->initialize()
 2 SYSTEMPATH\Database\BaseBuilder.php(1616): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `usuarios`
WHERE `usuario` = :usuario:', [...], false)
 3 SYSTEMPATH\Model.php(840): CodeIgniter\Database\BaseBuilder->get()
 4 APPPATH\Models\Users.php(21): CodeIgniter\Model->__call('get', [])
 5 APPPATH\Controllers\Login.php(24): App\Models\Users->obtenerUsuario('admin123')
 6 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Login->acceder()
 7 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Login))
 8 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-14 01:04:49 --> Class "App\Models\id_bancos" not found
in APPPATH\Controllers\Pago.php on line 31.
 1 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Pago->crear()
 2 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pago))
 3 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2023-06-14 01:21:25 --> mysqli_sql_exception: Unknown column 'banco' in 'field list' in C:\xampp\htdocs\proyectosistema\system\Database\MySQLi\Connection.php:295
Stack trace:
#0 C:\xampp\htdocs\proyectosistema\system\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pa...', 0)
#1 C:\xampp\htdocs\proyectosistema\system\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pa...')
#2 C:\xampp\htdocs\proyectosistema\system\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pa...')
#3 C:\xampp\htdocs\proyectosistema\system\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pa...', Array, false)
#4 C:\xampp\htdocs\proyectosistema\system\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
#5 C:\xampp\htdocs\proyectosistema\system\BaseModel.php(782): CodeIgniter\Model->doInsert(Array)
#6 C:\xampp\htdocs\proyectosistema\system\Model.php(730): CodeIgniter\BaseModel->insert(Array, true)
#7 C:\xampp\htdocs\proyectosistema\app\Controllers\Pago.php(118): CodeIgniter\Model->insert(Array)
#8 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(934): App\Controllers\Pago->guardar()
#9 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pago))
#10 C:\xampp\htdocs\proyectosistema\system\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#11 C:\xampp\htdocs\proyectosistema\index.php(67): CodeIgniter\CodeIgniter->run()
#12 {main}
CRITICAL - 2023-06-14 01:21:25 --> Unknown column 'banco' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 295.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(295): mysqli->query('INSERT INTO `pagos` (`id_proveedor`, `fecha_registro`, `forma_pago`, `num_cheque`, `num_transferencia`, `banco`, `fecha_movimiento`, `doc_adjunto`, `valor_total`) VALUES (\'4\', \'2023-06-09\', \'Cheque\', \'\', \'\', NULL, \'\', \'\', \'2222\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(691): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `pagos` (`id_proveedor`, `fecha_registro`, `forma_pago`, `num_cheque`, `num_transferencia`, `banco`, `fecha_movimiento`, `doc_adjunto`, `valor_total`) VALUES (\'4\', \'2023-06-09\', \'Cheque\', \'\', \'\', NULL, \'\', \'\', \'2222\')')
 3 SYSTEMPATH\Database\BaseConnection.php(605): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `pagos` (`id_proveedor`, `fecha_registro`, `forma_pago`, `num_cheque`, `num_transferencia`, `banco`, `fecha_movimiento`, `doc_adjunto`, `valor_total`) VALUES (\'4\', \'2023-06-09\', \'Cheque\', \'\', \'\', NULL, \'\', \'\', \'2222\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(2309): CodeIgniter\Database\BaseConnection->query('INSERT INTO `pagos` (`id_proveedor`, `fecha_registro`, `forma_pago`, `num_cheque`, `num_transferencia`, `banco`, `fecha_movimiento`, `doc_adjunto`, `valor_total`) VALUES (:id_proveedor:, :fecha_registro:, :forma_pago:, :num_cheque:, :num_transferencia:, :banco:, :fecha_movimiento:, :doc_adjunto:, :valor_total:)', [...], false)
 5 SYSTEMPATH\Model.php(330): CodeIgniter\Database\BaseBuilder->insert()
 6 SYSTEMPATH\BaseModel.php(782): CodeIgniter\Model->doInsert([...])
 7 SYSTEMPATH\Model.php(730): CodeIgniter\BaseModel->insert([...], true)
 8 APPPATH\Controllers\Pago.php(118): CodeIgniter\Model->insert([...])
 9 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Pago->guardar()
10 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pago))
11 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
12 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2023-06-14 01:24:49 --> Undefined variable $validacion
in APPPATH\Controllers\Pago.php on line 68.
 1 APPPATH\Controllers\Pago.php(68): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $validacion', 'C:\\xampp\\htdocs\\proyectosistema\\app\\Controllers\\Pago.php', 68)
 2 SYSTEMPATH\CodeIgniter.php(934): App\Controllers\Pago->guardar()
 3 SYSTEMPATH\CodeIgniter.php(499): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\Pago))
 4 SYSTEMPATH\CodeIgniter.php(368): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
